packages/auth/src/app.ts

```diff
@@ -342,12 +344,26 @@ export class App {
 
     koa.use(cors())
     koa.keys = [this.config.cookieKey]
+
+    const redis = await this.container.use('redis')
     koa.use(
       session(
         {
           key: 'sessionId',
           maxAge: 60 * 1000,
-          signed: true
+          signed: true,
+          store: {
+            async get(key) {
+              const data = await redis.get(key)
+              return data ? JSON.parse(data) : null
+            },
+            async set(key, session) {
+              await redis.set(key, JSON.stringify(session))
```

Comment:
How does expiry/TTL of the session get saved in redis?

Reply:
Testing the flow showed that it was being expired but it looks like it was just the http cookie. Redis key was not actually expiring. Changed it to make the record expire in redis. [05fcb02](https://github.com/interledger/rafiki/pull/2693/commits/05fcb02334c3cba30dceaf5d05472d59cfe5da3d)

Reply:
This does not set expiry on redis key. With set command, that would be done either via EX options of set command or through some expiry command. If we know the TTL of the session I would rewrite this code to be like this:
```
const op = redis.multi()
op.hmset(key, session)
op.expire(key, expirationInMilliseconds)

await op.exec()
```